components/homepage/HomeButton.tsx

```diff
@@ -7,9 +7,16 @@ interface ButtonProps {
 const HomeButton = (props: ButtonProps) => {
     return(
         <button className=" relative transition ease-in-out duration-500 delay-150 hover:scale-105 bg-white rounded-3xl w-20 h-[1.85rem] md:w-36 md:h-10 md:py-1.5 py-0.5 text-center z-50 overflow-hidden">
-            <Link href={props.link} target="_blank">
-                <h1 className="text-sm md:text-lg font-bold hover:text-cai-500 text-cai-400">{props.text}</h1>
-            </Link>
+            {/*If the link is within the website we'll use an anchor tag, otherwise we'll use the next/link component with a target attribute*/}
```

Comment:
This comment duplicates what the code shows below. 

Replace this comment so it explains why do we have to decide between `a` tag and `next/link` tag. Why does this nuance happen?